docs/source/athena-deploy.rst

```diff
@@ -0,0 +1,97 @@
+
+The name of the Lambda function is Athena Partition Refresh.
+
+The purpose of this function is to periodically refresh Athena tables to enable the search-ability of data.
+
+The default refresh interval is 10 minutes.
+
+Concepts
+--------
+
+This function utilizes:
+
+* `AWS S3 Event Notifications <http://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html>`_
+* `AWS SQS <https://aws.amazon.com/sqs/details/>`_
+* `AWS Lambda Invocations by Schedule <http://docs.aws.amazon.com/lambda/latest/dg/tutorial-scheduled-events-schedule-expressions.html>`_
+* `AWS Athena Repair Table <http://docs.aws.amazon.com/athena/latest/ug/ddl/msck-repair-table.html>`_
+
+Diagram
+~~~~~~~
+
+.. figure:: ../images/athena-refresh-arch.png
+  :alt: StreamAlert Athena Refresh Partition Diagram
+  :align: center
+  :target: _images/athena-refresh-arch.png
+
+Internals
+~~~~~~~~~
+
+Each time the Athena Partition Refresh Lambda function starts up, it does the following:
+
+* Polls the SQS Queue for the latest S3 event notifications (up to 50)
+* Deduplicates a set of S3 Bucket IDs from the notifications
+* Queries Athena to verify the ``streamalert`` database exists
+* Refreshes the Athena tables as configured below in the ``repair_type.repair_hive_table`` key
+* Deletes messages off the Queue once the Athena table(s) is successfully refreshed
+
+Getting Started
+---------------
+
+Configure Lambda Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Open ``conf/lambda.json``, and fill in the following ``Required`` options:
+
+
+===================================  ========  =========          ===========
+Key                                  Required  Default            Description
+-----------------------------------  --------  ---------          -----------
+``enabled``                          ``Yes``   ``True``           Enables/Disables the Athena Partition Refresh Lambda function
+``log_level``                        ``No``    ``info``           The log level for the Lambda function
+``memory``                           ``No``    ``128``            The amount of memory (in MB) allocated to the Lambda function
+``timeout``                          ``No``    ``60``             The maximum duration of the Lambda function (in seconds)
+``refresh_type.add_hive_partition``  ``No``    ``{}``             No currently supported
+``refresh_type.repair_hive_table``   ``Yes``   ``{}``             Key value pairs of S3 buckets and Athena table names.  Currently only supports the default alerts bucket created with every cluster.
+===================================  ========  =========          ===========
+
+**Example:**
+
+.. code-block:: json
+
+  {
+    "athena_partition_refresh_config": {
+      "enabled": true,
+      "log_level": "info",
+      "memory": 128,
+      "refresh_type": {
+        "add_hive_partition": {
+          "not_currently": "supported"
+        },
+        "repair_hive_table": {
+          "<prefix>.streamalerts": "alerts"
```

Comment:
This is the first mention of `<prefix>` - help the user better understand what this is and what they need to do